content/docs/instrumenting/exporters.md

```diff
@@ -124,6 +124,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [Pure Storage exporter](https://github.com/PureStorage-OpenConnect/pure-exporter)
    * [ScaleIO exporter](https://github.com/syepes/sio2prom)
    * [Tivoli Storage Manager/IBM Spectrum Protect exporter](https://github.com/treydock/tsm_exporter)
+   * [Azure storage usage exporter](https://github.com/PDOK/azure-storage-usage-exporter)
```

Comment:
Could you move it to the beginning of the list (to keep it sorted alphabetically)?